web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -1,5 +1,13 @@
 <template>
   <div>
+    <v-row
+      class="mx-2"
+      align="center"
+    >
+      <v-col cols="auto">
+        {{ meta.id }}
+      </v-col>
+    </v-row>
```

Comment:
Perhaps this section was added by accident? It results in the dandiset ID appearing above the main info panel.

```suggestion
```

Reply:
no it wasn't an accident. It was suggested by satra.

Reply:
@satra, could you explain what this is for?

Reply:
it is the full id of the dandiset (the metadata records have not all been updated to conform to the schema yet), but it is what it says, the id :)

Reply:
and yes, i wanted it there instead of as a row in the sections below.

Reply:
If you want it this way, then the PR is cleared for merge. But just so you know, it feels a lot like the webpage broke or something (like when unstyled content sometimes leaks onto the styled part of a page). My spider sense is telling me that doing it this way is a serious knock to the general UX of the site. Is there a more organic way to integrate the ID value into the DLP that we could explore in a followup?

Reply:
i see what you mean. yes it could look like this

**ID:** id

Reply:
i made a simple update for now. we can merge this and then plan a better UX overall design.